grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -410,8 +410,14 @@ class StockMovementService {
             if(params.createdBefore) {
                 le("dateCreated", params.createdBefore)
             }
+            def orderBy = "dateCreated"
 
-            order("dateCreated", "desc")
+            if(params.orderBy == "requisition.dateRequested") {
```

Comment:
this might not be the best solution for ordering by a nested value but I couldn't find a better one.

Reply:
I think other option here would be:
```
if (params.orderBy == "requisition.dateRequested") {
    requisition {
        order("dateRequested", desc)
    }
} else {
    order("dateCreated", "desc")
}

```
I think it has a slightly better readability.